### ISSUE TITLE
Left menu is broken since PR #67

### ISSUE THREAD

**Issue by gwleclerc** [vjeantet/hugo-theme-docdock#68]
If you go to https://docdock.netlify.com/, you can see that Left menu is broken du to modifications on partial menu.html

```html
<li data-nav-id="{{.URL}}" class="dd-item
    {{- if .IsAncestor $currentNode}}parent{{end}}
    {{- if eq .UniqueID $currentNode.UniqueID}}active{{end}}
    {{- if .Params.alwaysopen}}parent{{end -}}
">
```

This code has to consequence to render parent and active li elements as 

```html
<li data-nav-id="/getting-start/" class="dd-itemparentactive visited">
```

instead of 

```html
<li data-nav-id="/getting-start/" class="dd-item parent active visited">
```

I think that `-` in `{{}}` should not be used in class attribute of html elements.

**Comment by vjeantet** [comment]
fixed with commit 0343f50ffbc5140baeda1dc77ded13c49ed6bfde

**Comment by vjeantet** [comment]
thanks for reporting !
